src/helm/benchmark/run_expander.py

```diff
@@ -319,6 +319,15 @@ class ModelRunExpander(ReplaceValueRunExpander):
         "summarization_zs": ["openai/davinci", "openai/curie", "openai/text-davinci-002", "openai/text-curie-001"],
         "biomedical": ["openai/text-davinci-003"],  # TODO: add https://huggingface.co/stanford-crfm/BioMedLM
         "interactive_qa": ["openai/text-davinci-001", "openai/davinci", "ai21/j1-jumbo", "openai/text-babbage-001"],
+        "opinions_qa/openai": [
```

Comment:
I'd reserve the `/` for organization/model; so I'd do `opinions_qa_openai` and `opinions_qa_ai21`